.flake8

```diff
@@ -0,0 +1,31 @@
+[flake8]
+count = True
+# max-complexity should be 10
+max-complexity = 30
+# max-line-length should be 79 (pep8) or 88 (python/black) or 127 (GitHub editor)
+max-line-length = 370
```

Comment:
Why such a high limit? I'd expect it to be 80.

Reply:
This is the current state of the code.  Lines could be wrapped and this number lowered in future pull requests.